walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -27,35 +27,42 @@
   */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
 
-    ArrayList<TimeRange> allUnavailableTimes = new ArrayList<>();
+    ArrayList<TimeRange> mandatoryUnavailableTimes = new ArrayList<>();
+    ArrayList<TimeRange> optionalUnavailableTimes = new ArrayList<>();
 
-    for (Event event : events){
-      if (!isEventBlockingRequest(event, request)){
-        allUnavailableTimes.add(event.getWhen());
+    for (Event event : events) {
+
+      if (!isEventBlockingRequest(event, request, true)) {
```

Comment:
add `/* mandatory= */ true`

For any hard-coded parameter value like this, its good to include an inline description, since usually you can tell what you're passing in based on the variable name, but when its a hard-coded value you can't

Reply:
Okay, sounds good.